### ISSUE TITLE
Bugfix: isStatic

### ISSUE THREAD

**Issue by uhop** [tc39/proposal-decorators#7]
First of all, thank you for putting the proposal together.

Small bug to be fixed:

* This is how descriptors are presented in [METAPROGRAMMING.md](https://github.com/littledan/proposal-unified-class-features/blob/master/METAPROGRAMMING.md):
  ```js
  {
    kind: "method" or "field"
    key: String, Symbol or Private Name,
    placement: "static", "prototype" or "own",
    descriptor: Property Descriptor (argument to Object.defineProperty),
  }
  ```
* Later on `placement` is not used. Instead of it `isStatic` is used in several places:
  ```js
  memberDescriptor.extras = [{
    kind: "field",
    // The initializer callback is called with the receiver set to the new instance
    initializer() { return descriptor.value.bind(this); },
    descriptor,
    isStatic,
  }];
  ```

The intend is clear to me, but probably it should be fixed anyway.

**Comment by littledan** [comment]
Thanks for the bug report! I'm really happy to have your detailed review.

**Comment by littledan** [comment]
This was fixed in 21b1112195208e5a9db3a03a80d7a7d20d2241ff
